R/metapredict_process.R

```diff
@@ -73,58 +73,84 @@ fixGeoSampleNames = function(sampleNames) {
 
 
 fixCelSampleNames = function(sampleNames) {
-  sampleNamesNew = gsub('\\.cel(\\.gz)*$', '', sampleNames, ignore.case=TRUE)
+  sampleNamesNew = gsub('\\.cel(\\.gz)*$', '', sampleNames, ignore.case = TRUE)
   return(sampleNamesNew)}
 
 
 getGeneProbeMappingAffy = function(mappingFilePath) {
-  mapping = utils::read.table(mappingFilePath, sep='\t', header=TRUE, stringsAsFactors=FALSE)
-  mappingUnique = unique(mapping[,c('Probe.Set.Name', 'Affy.Probe.Set.Name')])
-  mappingUnique = mappingUnique[apply(mappingUnique, MARGIN=1, function(r) !any(is.na(r))),]
+  # mapping = utils::read.table(mappingFilePath, sep = '\t', header = TRUE, stringsAsFactors = FALSE)
+  mapping = fread(mappingFilePath, sep = '\t', header = TRUE, stringsAsFactors = FALSE)
+  mappingUnique = unique(mapping[,c('geneId', 'probeSet')])
```

Comment:
Actually, none of this seems to have changed from what we discussed on Friday. Is this your latest commit?